supervisor/utils/blockbuster.py

```
@@ -1,24 +1,62 @@
+        if cls._instance is None:
+            cls._instance = _BlockBuster()
+        cls._instance.activate()
+
+    @classmethod
+    def deactivate(cls):
+        """Deactivate blockbuster detection."""
+        _LOGGER.info("Deactivating BlockBuster blocking I/O detection")
+        if cls._instance:
+            cls._instance.deactivate()
+
 import logging
```

Ruff (0.11.9)

Module level import not at top of file

(E402)